quartz/cli/handlers.js

```diff
@@ -113,7 +113,7 @@ export async function handleCreate(argv) {
     }
   }
 
-  await fs.promises.unlink(path.join(contentFolder, ".gitkeep"))
```

Comment:
let's not silently swallow the error, we wrap this with an `fs.existsSync` instead

Reply:
Done. By the way, do you know why I may have encountered the no-`upstream` issue?

![image](https://github.com/jackyzha0/quartz/assets/417267/dc894b45-102b-4fc1-97ae-9870603ec4f9)

I see that it's mentioned in the Quartz 3 migration docs, but I'm starting fresh with Quartz 4. And in any case I already had `https://github.com/jackyzha0/quartz.git` as an upstream.

![image](https://github.com/jackyzha0/quartz/assets/417267/d13d7e11-4496-4985-9722-ae6ab7a7695c)

Reply:
By the way, I did some digging and it seems the `[fa](fatal: 'upstream' does not appear to be a git repository)` is a **normal** occurrence and can be ignored. In fact the command is using the failure of `git remote show` to decide whether to run `git remote add`.

![image](https://github.com/jackyzha0/quartz/assets/417267/1d978231-1087-415f-a679-a7281325006d)

It's only that as a first-time user of quartz I felt like something had gone wrong, when actually everything was working as expected. This was not obvious for either the `git remote show` or `unlink` warnings.

Reply:
thanks for the feedback! argh had no idea that bash still printed the error even with `||`, i'll take a look at it